ros2/nightly/nightly/core.yaml

```diff
@@ -0,0 +1,15 @@
+console_bridge: {ubuntu: []}
+fastcdr: {ubuntu: []}
+fastrtps: {ubuntu: []}
+libopensplice67: {ubuntu: []}
+libopensplice69: {ubuntu: []}
```

Comment:
Using an empty list to satisfy these will require anyone who does want to test with OpenSplice (or RTI Connext DDS below) to explicitly install the package as rosdep will resolve to the empty package.

Reply:
Do you know of a alternate way with could avoid installing these transport venders without this injection? Is this list overcompensating, or is there only a smaller set of keys we just need to skip?

> to explicitly install the package as rosdep will resolve to the empty package.

Users could also add a new files to `/etc/ros/rosdep/sources.list.d/` that takes president, no? Something like `10-default.list`, denoting priority like with udev rules? I thought of adding the the rosdep entries for the `prereqs.yaml` and `ros2.yaml` into a separate `default.list` file with higher priority, but then decided against deviating from the existing footprint of a conventional install for the sake of consistency.

Reply:
> Is this list overcompensating, or is there only a smaller set of keys we just need to skip?

The list in the documentation is a bit overstuffed to work for both Bouncy and Crystal. In general rosdep doesn't have the ability to distinguish between an optional or opportunistic dependency (use this this if it's there but don't freak out if it isn't) and a mandatory dependency. So I don't know that there's anything to do about it. It's just something that downstream consumers of the Dockerfile need to know if their intent is to use OpenSplice or RTI Connext DDS.

Reply:
Suppose we use `--skip-keys` for the opportunistic dependencies and keep only the mandatory dependencies in the prereqs.yaml, would that satisfy both uses? Short of calling rosdep install on a core package, rosdep wouldn't necessarily try to install the opportunistic dependency from a parent package already ignored/installed on the system.

Reply:
See commit ea0fdab